rpcs3/Emu/Audio/AudioDumper.cpp

```diff
@@ -1,13 +1,15 @@
 ﻿#include "stdafx.h"
 #include "AudioDumper.h"
 
+#include "Utilities/date_time.h"
+#include "Emu/System.h"
 
 AudioDumper::AudioDumper(u16 ch)
 	: m_header(ch)
 {
 	if (GetCh())
 	{
-		m_output.open(fs::get_cache_dir() + "audio.wav", fs::rewrite);
+		m_output.open(fs::get_cache_dir() + "audio_" + Emu.GetTitleID() + "_" + date_time::current_time_narrow<'_'>() + ".wav", fs::rewrite);
```

Comment:
For example like this:
```suggestion
		std::string path = fs::get_cache_dir() + "audio_";
		if (const std::string id = Emu.GetTitleID(); !id.empty())
		{
			path += id + "_";
		};
		path += date_time::current_time_narrow<'_'>() + ".wav";
		m_output.open(path, fs::rewrite);
```

Reply:
Thank you